src/planscape/forsys/generate_projects_for_a_single_scenario.R

```diff
@@ -1,88 +1,104 @@
 suppressMessages({
-  library(dplyr)
+  library(tidyverse)
   library(sf)
   library(forsys)
   library(patchmax)
   library(ggplot2)
   library(ggnewscale)
 })
 
-generate_projects_for_a_single_scenario <- function(forsys_input_data,
-                                                    priorities,
-                                                    priority_weights,
-                                                    stand_id_field,
-                                                    proj_id_field,
-                                                    stand_area_field,
-                                                    stand_cost_field,
-                                                    geo_wkt_field,
-                                                    output_scenario_name,
-                                                    output_scenario_tag) {
-  wp_str <- "weighted_priorities"
-  # Enables debug mode if output_scenario_name and output_scenario_tag are
-  # non-empty.
+generate_projects_for_a_single_scenario <- function(
+  forsys_input_data,
+  priorities,
+  priority_weights,
+  stand_id_field,
+  proj_id_field,
+  stand_area_field,
+  stand_cost_field,
+  geo_wkt_field,
+  output_scenario_name,
+  output_scenario_tag,
+  PreForsysClusterType = 'KMEANS') {
```

Comment:
Can this be a boolean value?
e.g. enable_kmeans_clustering = FALSE to start

The plumbing work I've done in #597 disables all clustering by default.